challenges/CC_30_phyllotaxis_p5.js/sketch.js

```diff
@@ -26,7 +27,7 @@ function draw() {
     var x = r * cos(a);
     var y = r * sin(a);
     var hu = sin(start + i * 0.5);
-    hu = map(hu, -1, 1, 0, 360);
+    hu = map(hu, -1, 1, colorRange[0], colorRange[1]);
```

Comment:
See the p5.js reference: "Setting colorMode(HSB) lets you use the HSB system instead. By default, this is colorMode(HSB, 360, 100, 100, 1). You can also use HSL."